code/modules/power/cells/device_cells.dm

```diff
@@ -63,4 +88,17 @@
 	origin_tech = list(TECH_POWER = 8, TECH_ENGINEERING = 6)
 
 /obj/item/weapon/cell/device/weapon/recharge/alien/update_icon()
-	return // No overlays please.
\ No newline at end of file
+	return // No overlays please.
+
+//YAWN Addtion
+/obj/item/weapon/cell/device/weapon/recharge/alien/omni
+	name = "omni weapon power cell"
+	desc = "A mix bettewn alien technology and phoron tech. Seems to fit in almost any cell slot..."
+	charge_amount = 90 // 5%.
+	maxcharge = 1800
+	charge_delay = 500 // Every 50 seconds.
```

Comment:
```suggestion
	charge_delay = 50 SECONDS
```
Generally, using the time macros makes for more easily read time (rather than having to remember that everything is in deciseconds) :P